__tests__/classes/InputFormatter.js

```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line no-unused-vars
+/* globals describe, beforeAll, afterAll, beforeEach, afterEach, test, expect */
+'use strict'
+
+const InputFormatter = require('../../classes/InputFormatter')
+
+describe('inputFormatter', () => {
+  describe('format string to array', () => {
+    test('should return array splitted by given separator', () => {
+      const input = 'value1+value2'
+      const separator = '+'
+      const formatted = InputFormatter.formatStringToArray({ input, separator })
+      expect(formatted.length).toBe(2)
+      expect(formatted).toEqual(expect.arrayContaining(['value1', 'value2']))
```

Comment:
no need for length check and containing check - could just use `toEqual`
```suggestion
      expect(formatted).toEqual(['value1', 'value2'])
```